config/serverless.oblt.yml

```diff
@@ -27,3 +27,13 @@ xpack.apm.serverlessOnboarding: true
 xpack.fleet.packages:
   - name: apm
     version: latest
+## Disable APM UI components and API calls
+xpack.apm.featureFlags.agentConfigurationAvailable: false
+xpack.apm.featureFlags.configurableIndicesAvailable: false
+xpack.apm.featureFlags.infrastructureTabAvailable: false
+xpack.apm.featureFlags.infraUiAvailable: false
+xpack.apm.featureFlags.migrationToFleetAvailable: false
+xpack.apm.featureFlags.sourcemapApiAvailable: false
+xpack.apm.featureFlags.spacesAvailable: false
+xpack.apm.featureFlags.schemaAvailable: false
```

Comment:
When do we want `schemaAvailable` enabled and not `migrationToFleetAvailable`? And vice versa?

Reply:
After merging https://github.com/elastic/kibana/pull/158911, I need to check if we are still using `schemaAvailable`, I think we just need `migrationToFleetAvailable`, as @ogupte [commented](https://github.com/elastic/kibana/pull/158911#discussion_r1221482946) in the other PR, the tab for the fleet migration is called Schema, but it's all related with fleet migration

Reply:
Remove it here https://github.com/elastic/kibana/pull/159136/commits/eca126ce3d4c6463b6219b7c84c524e92a88da97